README.md

```diff
@@ -1,18 +1,89 @@
 
-These are the scripts I use while releasing [The Foreman](https://theforeman.org). They automate most of the [release process](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/release.md.erb).
+A collection of small scripts, adhering to the [Unix philosophy](https://en.wikipedia.org/wiki/Unix_philosophy), used to release [The Foreman](https://theforeman.org). They automate parts of the [branch](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/branch.md.erb) and [release](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/release.md.erb) processes. It is also used for the [Katello](https://theforeman.org/plugins/katello/) [branch](https://github.com/theforeman/tool_belt/blob/master/procedures/katello/branch.md.erb) and [release](https://github.com/theforeman/tool_belt/blob/master/procedures/katello/release.md.erb).
+
+The most important environment variables are `PROJECT` and `VERSION`.
+
+## Procedures
+
+Procedures are kept in [tool_belt](https://github.com/theforeman/tool_belt) with small wrappers in this repository.
+
+### Roles
+
+Both the branch and release proceduress reconize an owner and engineer. Someone can be both owner and engineer at the same time, and it is preferable for a fast process. Having both in (very) different timezones can lengthen the process. Below is a listing of access.
+
+#### Foreman release owner
+
+* Belong to the [Foreman GitHub release-managers team](https://github.com/orgs/theforeman/teams/release-managers)
+
+#### Foreman release engineer
+
+* Belong to the [Foreman GitHub release-engineering team](https://github.com/orgs/theforeman/teams/release-engineering)
+* [Foreman infrastructure access](https://theforeman.github.io/foreman-infra/#access)
+* [Jenkins access](https://theforeman.github.io/foreman-infra/jenkins/#access)
+* [Discourse](https://community.theforeman.org/) account level 3 to post to [Release announcements](https://community.theforeman.org/c/release-announcements/8)
+* IRC: permission to change the topic in [#theforeman on Libera.Chat)[ircs://irc.libera.chat/theforeman)
+
+#### Katello release owner
+
+* Write permission to [Katello/katello](https://github.com/Katello/katello)
+* Push access to the [Katello gem](https://rubygems.org/gems/katello)
+* [Discourse](https://community.theforeman.org/) account level 3 to post to [Release announcements](https://community.theforeman.org/c/release-announcements/8)
+
+#### Katello release engineer
+
+These are the same as Foreman's release engineer.
+
+### Branching
+
+Use `procedure_branch` to display the steps. Modify `PROJECT` and `VERSION` as needed.
+
+```sh
+PROJECT=foreman VERSION=3.7 ./procedure_branch
+```
+
+It will have created `releases/${PROJECT}/${VERSION}/settings`, which should be submitted as a pull request.
+
+A more advanced invocation is to specify the parameters directly and copy it. This uses `wl-copy` for Wayland; `xclip` on Xorg achieves the same.
```

Comment:
s/it./them./

Reply:
With `it` I meant the output, not the parameters. I'll change that to be more explicit.